rolepermissions/roles.py

```diff
@@ -172,6 +176,13 @@ def get_default(cls, permission_name):
         return cls.available_permissions[permission_name]
 
 
+def get_or_create_permission(permission_name):
+    """Get a Permission object from a permission name."""
+    user_ct = ContentType.objects.get_for_model(get_user_model())
+    return Permission.objects.get_or_create(
+            content_type=user_ct, codename=permission_name, name=camel_or_snake_to_title(permission_name))
```

Comment:
I think `name` would be better as a `defaults` parameter. This way it would be possible to change it manually after creation without the risk of breaking the functionality.

Reply:
Can you also write a test for this behavior?

Reply:
Oh!  That's such a good idea!  thanks.
Yes - sorry, didn't see that was missing.

Reply:
I see -- this code would cause problems when looking up existing permissions whose name were set by some other method (or not at all, e.g.)  Will fix and add test cases for backwards compat.